src/webchat-ui/components/presentational/DisconnectOverlay.tsx

```diff
@@ -1,78 +1,97 @@
-import React from 'react';
-import styled from '@emotion/styled';
-import IconButton from './IconButton';
-import CloseIcon from '../../assets/baseline-close-24px.svg';
-import Button from './Button';
+import React from "react";
+import styled from "@emotion/styled";
+import IconButton from "./IconButton";
+import CloseIcon from "../../assets/baseline-close-24px.svg";
+import Button from "./Button";
 
 const Wrapper = styled.div(({ theme }) => ({
-    display: 'flex',
-    justifyContent: 'center',
-    alignItems: 'center',
+	"@keyframes appearAnimation": {
+		from: {
+			opacity: 0,
+			transform: "scale(.95)",
+			translateZ: -2,
+		},
+		to: {
+			opacity: 0.93,
+			transform: "scale(1)",
+			translateZ: -2,
+		},
+	},
 
-    position: 'absolute',
-    left: 0,
-    top: 0,
-    width: '100%',
-    height: '100%',
+	animation: "appearAnimation 0.33s 0.75s both",
+	display: "flex",
+	justifyContent: "center",
+	alignItems: "center",
 
-    padding: theme.blockSize,
-    boxSizing: 'border-box',
-    backgroundColor: 'hsla(0, 0%, 0%, .33)',
+	position: "absolute",
+	left: 0,
+	top: 0,
+	width: "100%",
+	height: "100%",
 
-    zIndex: 2,
+	padding: 20,
+	boxSizing: "border-box",
+	backgroundColor: `color-mix(in srgb, ${theme.backgroundWebchat}, #0001)`,
```

Comment:
I think we can just use theme.white here
backgroundWebchat is actually not the color for the chat history background, it's going on the header for example (see my open PR). I adjusted this according to the Figma designs (though they are a bit confusing in regards to usage of this color)